ingest/de.py

```diff
@@ -170,13 +170,54 @@ def execute_de(self):
                 self.cluster_name,
                 self.method,
             )
-            DifferentialExpression.de_logger.info("preparing DE on dense matrix")
         else:
-            msg = f"Submitted matrix_file_type should be \"dense\" or \"mtx\" not {self.matrix_file_type}"
+            msg = f"Submitted matrix_file_type should be \"dense\" or \"mtx\" not \"{self.matrix_file_type}\""
+            log_exception(
+                DifferentialExpression.dev_logger, DifferentialExpression.de_logger, msg
+            )
```

Comment:
Does this log to Mixpanel?  Such observability would be helpful.

Reply:
Sadly no. This merely records it in user_log.txt and log.txt
Mixpanel work is ticketed in SCP-4206